installation_instruction/get_flags_and_options_from_schema.py

```diff
@@ -46,12 +46,14 @@ def _get_flags_and_options(schema: dict, misc: dict = None) -> list[Option]:
         option_default = value.get('default', None)
 
         if "enum" in value:
-            option_type = Choice( value["enum"] )
+            option_type = Choice(value["enum"])
+        elif option_type == 'array' and value.get('items', {}).get('enum'):
+            option_type = Choice(value['items']['enum'], case_sensitive=False, multiple = True)
```

Comment:
My "suggestion" mentioning the error did not show.

The problem here is that `Choice` das not have the property `multiple`. `Option` has said property.